docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+for each report, but does not enforce any uniqueness constraint on the ``title`` of reports.
+
+Alternatively, we can update an existing report, adding more IOCs and/or replacing existing ones.  Note that we have
+to find a report associated with a watchlist by looking in its ``reports`` collection.
+
+Adding the Report to a Watchlist
+++++++++++++++++++++++++++++++++
+Now, add the new Report to a new Watchlist:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Watchlist
+    >>> watchlist = Watchlist(api, None, {"alerts_enabled": False,
+    ...                                   "classifier": None,
+    ...                                   "description": "We are on the lookout for any signs of suspicious applications running on our endpoints",
+    ...                                   "name": "Suspicious Applications",
+    ...                                   "report_ids": [report.id],
+    ...                                   "tags_enabled": True})
+    >>> watchlist.save()
+
+**Note:** ``classifier`` should always be ``None``, unless the new watchlist is being subscribed to a feed.
+
+When either ``alerts_enabled`` or ``tags_enabled`` are ``True``, that Watchlist will create data you can act on -
+either alerts or hits, respectively; if both are ``False``, the Watchlist is effectively disabled.
+
+If you already have an existing Watchlist you wish to enhance, you can add Reports to the existing Watchlist.
+
+Enabling Alerting on a Watchlist
+++++++++++++++++++++++++++++++++
+Once you have the Watchlist configured with the IOCs that are generating the kinds of hits (results) you are after,
+you can enable Alerting for the Watchlist.
+
+::
+
+    >>> watchlist.enable_alerts()
+
+A Closer Look at IOCs
+---------------------
+In this document, we will only discuss the "v2" IOCs; other types of IOCs are only provided for backwards compatibility
+reasons, and are converted, internally, to this type.
+
+IOCs can be classified into two general types, depending on their ``match_type`` value:
+
+*Query IOCs* are those with a ``match_type`` of ``query``; their ``values_list`` contains a single string that
+specifies a query compatible with process searches.  For example, the following IOC looks for the process ``git.exe``
+that does *not* connect to one of a specified list of IP addresses:
+
+::
+
+    {
+        "id": "example_1",
+        "match_type": "query",
+        "values": ["process_name:git.exe NOT (netconn_ipv4:35.158.151.206 OR netconn_ipv4:1.1.244.78
+                    OR netconn_ipv4:80.18.61.229 OR netconn_ipv4:80.18.61.228)"]
+    }
+
+Query IOCs are searched every 5 minutes by the Carbon Black Cloud, and are tested against a rolling window of the
+last hour's worth of data for the organization.  (They will *not* generate hits or alerts for process attributes that
+were reported more than an hour in the past.)  They may employ any searchable field, and may employ complex query
```

Comment:
Link to the searchable fields

Reply:
Where can I find those?

Reply:
I want you to try to find it given our documentation. Though I will say that it is on developer.carbonblack.com

Reply:
This is the only page we've got for that: https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/platform-search-fields/

Reply:
Looks like the right page to me

Reply:
And which fields on that big list are "fair game" to use here? (I don't like that page; it's very confusing to me.)

Reply:
Have you checked the section at the top `Using the Schema`

Reply:
Added a link in.